app/Home/Home.module.css

```diff
@@ -0,0 +1,22 @@
+.container {
+  width: full;
+  max-width: 1024px;
+  height: 100vh;
+  margin: 0 auto;
+  padding: 6rem;
```

Comment:
question: Why are you using rem here but at line 3 you used px? How do you decide when to use one or another?

Reply:
rem is supposed to be responsive design?

Reply:
px is for fixed width